### DISCUSSION TITLE
[Feature Request] C# SDK

### DISCUSSION THREAD

**Discussion by msyzzm** [zkSync-Community-Hub/zksync-developers#41] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

Need C# SDK to use in our backend.

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

developers

**Comment by MexicanAce** [comment]
Hi @msyzzm 👋 !

This is not currently in our pipeline, but we're open to review any public repos that want to provide a community supported SDK (like C#).
